v21.2/use-cloud-storage-for-bulk-operations.md

```diff
@@ -225,6 +225,87 @@ A custom root CA can be appended to the system's default CAs by setting the `clo
 
 </section>
 
+## Storage permissions
+
+This section describes the minimum permissions required to run CockroachDB bulk operations. While we outline information on the necessary permissions to access Amazon S3 and Google Cloud Storage when running these operations, the provider's documentation provides detail on the setup process and different options regarding access management.
+
+Depending on the actions a bulk operation performs, it will require different access permissions to a cloud storage bucket. 
+
+This table outlines the actions that each operation performs against the storage bucket:
+
+Operation                   | Permission                   | Description                                                                     
+----------------------------+------------------------------+---------------------------------------------------
+[Backup](backup.html)       | Write<br><br>Get<br><br>List<br><br>Delete | Write: Backups write the backup data to the bucket/container. During a backup job, a `BACKUP CHECKPOINT` file will be written that tracks the progress of the backup.<br>Get: Backups need get access after a [pause](pause-job.html) to read the checkpoint files on [resume](resume-job.html).<br>List: Backups need list access to the files already in the bucket. For example, `BACKUP` uses list to find previously taken backups when executing an incremental backup.<br> Delete: Backups may delete files and progress during the job after a pause in order to resume the job from a certain point. This can include overwriting backup work that had happened before the pause.
+[Restore](restore.html)     | Get<br>List          | Get: Restores need access to retrieve files from the backup.<br>List: During a restore job, it is necessary to pass the backup's directory. This contains metadata files that describe the backup as well as other versioned subdirectories and files. Restore also requires access to the `LATEST` file in order to read the latest available backup.
```

Comment:
non-blocking: But I think that the `List` description here sounds more like why we need Get.  If you wanted, the reasons we need list here are similar to why we need list in Backup, namely to find other backups in the backup collection.